src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -1154,23 +1199,27 @@ const bool ManchesterpatternDecoder::doDecode() {
 			}
 
 
-
-#			ifdef DEBUGDECODE
-			if (pdec->pattern[pdec->message[i]] < 0)
-				value = (value + 0x20); //lowwecase
-			Serial.print(value);
-			#endif
-
-			if (hasbit) {
-				ManchesterBits.addValue((pdec->pattern[pdec->message[i]] > 0 ? 1 : 0));
-				#ifdef DEBUGDECODE
-				Serial.print(ManchesterBits.getValue(ManchesterBits.valcount-1));
-				#endif;
-				hasbit = false;
+			if (mc_start_found) { // don't write if manchester processing was canceled
```

Comment:
wird in Zeile 1125 bereits geprüft. Kann doch eigentlich hier nie false sein oder?